terminal/src/com/jediterm/terminal/ui/TerminalPanel.java

```diff
@@ -1119,16 +1152,6 @@ private void drawCharacters(int x, int y, TextStyle style, CharBuffer buf, Graph
       }
     }
 
-    Color backgroundColor = getPalette().getColor(myStyleState.getBackground(style.getBackgroundForRun()));
```

Comment:
Why do you remove this block of code? It clears the rectangle for a char to be printed.